packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +598,48 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (options.useSlasPrivateClient) {
+            const clientSecret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+            if (!clientSecret) {
+                app.use('/ssr/auth', (_, res) => {
+                    return res.status(501).json({
+                        message:
+                            'Environment variable SLAS_PRIVATE_CLIENT_ID not set: LINK_TO_DOC HERE'
+                    })
+                })
+            } else {
+                const clientId = options.mobify.app.commerceAPI.parameters.clientId
+                const shortCode = options.mobify.app.commerceAPI.parameters.shortCode
```

Comment:
I am reading these from the config file. Should these be environment variables instead?

Reply:
Yes they should – but that doesn't nessecarily mean that you shouldn't read them from the config file.

**As part of this change, we should allow the client ID to be defined in an environment variable.**

Right now we recommend folks keep a mapping of API clients in code and load the right one ...

https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/proxying-requests.html#look-up-the-current-proxy-configuration

This is super awkward and means if you ever want to connect to another instance (or swap clients) you need to make a code change.

Reply:
I agree that it's a good idea (and could help ease switching between a public vs private client) but I think it should be in a separate PR. It's a separate feature from SLAS private clients

Reply:
If it hasn't been done already, we should create a WI to track that work